boards/MicroBit/microbit_zfp.gpr

```diff
@@ -96,6 +96,7 @@ library project MicroBit_ZFP is
     Src_Dirs_Root & "/middleware/src/bitmap", -- From middleware config
     Src_Dirs_Root & "/middleware/src/command_line", -- From middleware config
     Src_Dirs_Root & "/middleware/src/sdmmc", -- From middleware config
+    Src_Dirs_Root & "/middleware/src/neopixel", -- From middleware config
```

Comment:
Can you run the `boards/board_projects_generator.py` script to re-generate projects for all boards?